common/pom.xml

```diff
@@ -63,6 +63,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
@jmfinelli please can you point out where this is used in the build after this change?

Reply:
The test jar gets extracted [here](https://github.com/jbosstm/narayana/pull/2049/files#diff-d4f42289a362c7d991ef7305ebf659ea59d80c9fcb0e3ee42fc80f79f457aa92R98)

Reply:
Although I think your link is not going to a location that I can find, I do see the use of `>common<` for `<classifier>test-sources</classifier>` in qa/pom.xml but I can't see `<classifier>sources</classifier>` - is it definitely needed here?

Reply:
For a bit of clarity on the specific xml I am querying:
```
          <execution>
            <id>attach-sources</id>
            <goals>
              <goal>jar-no-fork</goal>
            </goals>
          </execution>
```
That applies for `jts` comment too in it's respective pom.xml.

Reply:
I can find some unzipping in https://github.com/jbosstm/narayana/blob/20ada636f438ae8bbaf550b10c38490bc5384b23/qa/tests/build.xml but I don't think that `>jts<` or `>common< ` `sources` (rather than `test-sources`) are used in there but perhaps the reason you are providing these is to keep it generating sources for these modules as well as the test sources? I am not sure so clarification would be great, thank you.

Reply:
> ... perhaps the reason you are providing these is to keep it generating sources for these modules as well as the test sources? I am not sure so clarification would be great, thank you.

+1 that is indeed the reason :-)

Reply:
Thank you for confirming!